docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,116 @@
+You can also use the slider to adjust the ESC input value.
+
+
+================================
+Live Plotting and Active metrics
+================================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_command_example.png
+One main feature of Vertiq Testing Tool is the ability to plot any supported metric. For example, let's command 3 volts to the module using the Control Panel in the Voltage Input tab.
+
+**WARNING: Please make sure there are no propellers attached to your module before attempting to spin it.**
+
+Set the 'Target Voltage' to 3.00 and click "Send Command". The module will ramp up to 3 volts over the next 5 seconds specified by the 'Ramp Time' value.
+After 5 seconds, the module is now constantly spinning at 3 volts. Let's confirm that this is true by plotting the drive volts.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/drive_volts_metric_plotting_example.png
+
+To visualize that 3 volts are indeed being applied to the module navigate to the widget labeled 'Select a metric to plot'.
+In the upper dropdown menu, select *brushless_drive*. This corresponds to the :ref:`Brushless Drive <brushless_drive>` client.
+In the lower dropdown menu, select *drive_volts*. This corresponds to the drive_volts client entry under Brushess Drive.
+Then click 'Update Plot'. 
+
+Notice that the *drive_volts* metric is added to the 'Active metrics' widget. By default, *obs_velocity*, *watts*, and *amps* are preloaded in the 'Active metrics' widget.
+Selecting a metric will activate it and update the live plot. Currently, only one metric can be active at a time, but there is no limit to the number of metrics that can be added to the 'Active metrics' widget.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/remove_metric_example.png
+To remove a metric from the 'Active metrics' widget, simply right click on it and select 'Remove metric'.
+
+
+===============================
+Auto Command and Ramp to Target
+===============================
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input.png
+
+Another useful feature found in the Voltage Input, Velocity Input, and PWM Input tabs is 'Ramp to target' in the Control Panel.
+This feature is enabled by default, which means when you click 'Send Command' after setting the desired Target Voltage, the module will attempt to ramp to the target voltage in increments. 
+The increments are determined by the ramp time, so increasing the ramp time will make it so that the module takes longer to reach the target voltage.
+
+The 'Auto Command' feature is disabled by default. When enabled, you will not have to click the 'Send Command' button to send a voltage command. 
+Instead, the application will automatically send the command whenever the target voltage value changes.
+
+Be very cautious when 'Auto Command' is enabled and 'Ramp to Target' is disabled. It is recommended to lower the 'Step Amount' to avoid large jumps between each target voltage command.
+
```

Comment:
A screen capture of what it looks like to do a slow v fast ramp v no ramp (or just screen shots even of what each looked like) would be very nice. It could also be used to show the difference between auto and manual sending commands.